public/views/modals/tx-details.html

```diff
@@ -99,7 +99,7 @@ <h4 class="title m0" translate>Details</h4>
       <span class="right">{{btx.feeStr + (exchangeRates.GBYTE_USD ? ' (≈ $'+(btx.fee * exchangeRates.GBYTE_USD / 1e9).toLocaleString([], {maximumFractionDigits: 5})+')' : '') }}</span>
     </li>
     <li class="line-b p10 row collapse">
-      <div class="text-gray small-2 columns"><span>Unit</span>:</div>
+      <div class="text-gray small-2 columns"><span translate>Unit</span>:</div>
```

Comment:
please don't translate Unit, I'm afraid attempts to translate it will be confusing rather than helpful

Reply:
But it's rather odd to have a fully translated wallet and then one string is not translated.
Also, why not call it just TX or Unit ID because more people will understand what is that than Unit?

Reply:
Also, just Unit conflicts with Unit sizes like: gigabyte, megabyte, kilobyte, byte.